app/schemas/models/user.coffee

```diff
@@ -145,125 +145,14 @@ _.extend UserSchema.properties,
-        type: 'string'
-      phoneScreenFilter: phoneScreenFilter
-      schoolFilter: schoolFilter
-      locationFilter: locationFilter
-      roleFilter: roleFilter
-      seniorityFilter: seniorityFilter
-      featured:
-        title: 'Featured'
-        type: 'boolean'
-        description: 'Should this candidate be prominently featured on the site?'
-  jobProfileApproved: {title: 'Job Profile Approved', type: 'boolean', description: 'Whether your profile has been approved by CodeCombat.'}
-  jobProfileApprovedDate: c.date {title: 'Approved date', description: 'The date that the candidate was approved'}
-  jobProfileNotes: {type: 'string', maxLength: 1000, title: 'Our Notes', description: 'CodeCombat\'s notes on the candidate.', format: 'markdown' }
-  employerAt: c.shortString {description: 'If given employer permissions to view job candidates, for which employer?'}
-  signedEmployerAgreement: c.object {},
-    linkedinID: c.shortString {title: 'LinkedInID', description: 'The user\'s LinkedIn ID when they signed the contract.'}
-    date: c.date {title: 'Date signed employer agreement'}
-    data: c.object {description: 'Cached LinkedIn data slurped from profile.', additionalProperties: true}
-  savedEmployerFilterAlerts: c.array {
-    title: 'Saved Employer Filter Alerts'
-    description: 'Employers can get emailed alerts whenever there are new candidates matching their filters'
-  }, c.object({
-    title: 'Saved filter set'
-    description: 'A saved filter set'
-    required: ['phoneScreenFilter','schoolFilter','locationFilter','roleFilter','seniorityFilter','visa']
-  }, {
-    phoneScreenFilter:
-      title: 'Phone screen filter values'
-      type: 'array'
-      items:
-        type: 'boolean'
-    schoolFilter:
-      title: 'School filter values'
-      type: 'array'
-      items:
-        type: schoolFilter.type
-        enum: schoolFilter.enum
-    locationFilter:
-      title: 'Location filter values'
-      type: 'array'
-      items:
-        type: locationFilter.type
-        enum: locationFilter.enum
-    roleFilter:
-      title: 'Role filter values'
-      type: 'array'
-      items:
-        type: roleFilter.type
-        enum: roleFilter.enum
-    seniorityFilter:
-      title: 'Seniority filter values'
-      type: 'array'
-      items:
-        type: roleFilter.type
-        enum: seniorityFilter.enum
-    visa:
-      title: 'Visa filter values'
-      type: 'array'
-      items:
-        type: visa.type
-        enum: visa.enum
-  })
+  # Deprecated. TODO: 1. remove user data 2. remove these fields
+  jobProfile: c.object { type: 'object', description: 'deprecated' }
+  jobProfileApproved: { type: 'boolean', description: 'deprecated' }
+  jobProfileApprovedDate: c.date { description: 'deprecated' }
+  jobProfileNotes: { type: 'string', description: 'deprecated' }
+  employerAt: c.shortString {description: 'deprecated'}
+  signedEmployerAgreement: c.object { type: 'object', description: 'deprecated' }
```

Comment:
This should be like `jobProfileApproved`, since this would fail to validate for anyone with a non-empty object set for this property.